### ISSUE TITLE
OCI purl's don't follow the spec

### ISSUE THREAD

**Issue by mattmoor** [chainguard-dev/apko#281]
I see the following in the `apko` `purl`s:
```
pkg:oci/ghcr.io%2Fdistroless%2Fstatic:latest-20220710@sha256:96556b7e982dbca4c582b5bee35ff205d2e1404a616e8f1f822c0def27b5ac3f?arch=amd64&tag=latest-20220710
```

According to [the spec](https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst#oci) this should be:
```
pkg:oci/static@sha256:96556b7e982dbca4c582b5bee35ff205d2e1404a616e8f1f822c0def27b5ac3f?repository_url=ghcr.io%2Fdistroless%2Fstatic&arch=amd64&tag=latest-20220710
```

cc @puerco

**Comment by kaniini** [comment]
Yeah I was wondering about that, if puerco doesn’t get to it tomorrow I can fix it.

**Comment by kaniini** [comment]
Fixed by #283.
